TombEngine/Renderer/RendererDraw.cpp

```diff
@@ -610,14 +611,129 @@ namespace TEN::Renderer
 		}
 	}
 
+	void Renderer::DrawFishSwarm(RenderView& view, RendererPass rendererPass)
+	{
+		using namespace TEN::Entities::Creatures::TR3;
+
+		if (!Objects[ID_FISH_EMITTER].loaded)
+			return;
+
+		if (rendererPass == RendererPass::CollectTransparentFaces)
+		{
+			for (const auto& fish : FishSwarm)
+			{
+				if (fish.Life > 0.0f)
```

Comment:
Can we invert this condition to `if (fish.Life <= 0.0f) continue;` to avoid excessive tabulation?

Reply:
fixed